apps/src/redux/levelProperties.js

```diff
@@ -0,0 +1,76 @@
+var _ = require('lodash');
+
+var SET_INITIAL_LEVEL_PROPS = 'levelProperties/SET_INITIAL_LEVEL_PROPS';
+
+// TODO - use makeEnum so we dont need to do indexOf?
+var ALLOWED_KEYS = [
+  'assetUrl',
+  'channelId',
+  'isDesignModeHidden',
+  'isEmbedView',
+  'isReadOnlyWorkspace',
+  'isShareView',
+  'isProjectLevel',
+  'isSubmittable',
+  'isSubmitted',
+  'isViewDataButtonHidden',
+  'instructionsMarkdown',
+  'instructionsInTopPane',
+  'puzzleNumber',
+  'stageTotal',
+  'showDebugButtons',
+  'showDebugConsole',
+  'showDebugWatch',
+];
+
+var levelInitialState = {
+  assetUrl: function () {}
+};
+
+ALLOWED_KEYS.forEach(function (key) {
+  levelInitialState[key] = levelInitialState[key] || undefined;
+});
```

Comment:
I did this to be consistent to what we had before. It makes it so that if you do `reduxStore.getState().level` it will show you all these props, including those that were never set. I'm not sure there's any good reason to do that though really. Probably makes sense to axe this chunk.